changelogs/unreleased/gh-9849-grant-revoke-user-right-after-upgrade.md

```diff
@@ -1,2 +1,6 @@
 ## feature/box
 * Added `on_schema_upgrade_end` trigger (gh-9849).
+
+## bugfix/box
+* User rights are now granted/revoked after upgrading automatically without
+  restarting (gh-9849).
```

Comment:
Two headers within one changelog entry are not parsed correctly. Check out the `tools/gen-release-notes` output. You can run it locally or look on the `lint/release-notes` CI job.

Also, we usually separate the header from the body with a blank line (the tool understands it anyway, but it is for readability).

Reply:
Added a blank line.
Removed the second entry because the trigger had been deleted.